### PR TITLE
Issue #243 About page

### PR DESCRIPTION
**Pull request by kubili**
I think i wrote and linked everything important. #243

### CODE REVIEW

File: piplmesh/frontend/templates/about.html [hunk 1]
```diff
@@ -5,6 +5,20 @@
...
 
 {% block content %}
-    <h1>{% trans "About" %}</h1>
-    <p>TODO.</p>
-{% endblock %}
\ No newline at end of file
+    <div>
+        <h1>{% trans "About" %}</h1>
+
+        <p>{% blocktrans %}PiplMesh is the project of students at <a class="ext-link" href="http://www.fri.uni-lj.si/">Faculty of Computer and Information Science</a> of University of Ljubljana and <em>wlan slovenija</em> open wireless network. Its main goal is the development of an open source social networking and info portal for wireless networks used as a  welcome page when users connect to the network.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Double space before "welcome page".

File: piplmesh/frontend/templates/about.html [hunk 2]
```diff
@@ -5,6 +5,20 @@
 {% block title %}{% trans "About" %}{% endblock %}
 
 {% block content %}
-    <h1>{% trans "About" %}</h1>
-    <p>TODO.</p>
-{% endblock %}
\ No newline at end of file
+    <div>
+        <h1>{% trans "About" %}</h1>
+
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Unnecessary empty lines.

File: piplmesh/frontend/templates/about.html [hunk 3]
```diff
@@ -5,6 +5,20 @@
...
+        <p>{% blocktrans %}You can read more about the project on  <a class="ext-link" href="http://dev.wlan-si.net/wiki/PiplMesh/">development pages</a> of wlan slovenija.{% endblocktrans %}</p>
+
+        <p>{% blocktrans %}For development <a class="ext-link" href="https://github.com/">GitHub</a> is used, so source code and issue tracker is found there. A list of contributors is also avaible as <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/stargazers/">GitHub profiles list</a>.{% endblocktrans %}</p>
+
+        <p>{% blocktrans %} Project is developed under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. See the <a class="ext-link" href="http://www.gnu.org/licenses/">GNU Affero General Public License</a> for more details.{% endblocktrans %}</p>
+
+        <p>{% blocktrans %}If you have found a bug or have any other technical issue with the site, please use <a href="https://github.com/wlanslovenija/PiplMesh/issues?state=open">GitHub to report it</a>.
+
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: No breaks in `blocktrans`.

File: piplmesh/frontend/templates/about.html [hunk 4]
```diff
@@ -5,6 +5,20 @@
...
+        <p>{% blocktrans %}You can read more about the project on  <a class="ext-link" href="http://dev.wlan-si.net/wiki/PiplMesh/">development pages</a> of wlan slovenija.{% endblocktrans %}</p>
+
+        <p>{% blocktrans %}For development <a class="ext-link" href="https://github.com/">GitHub</a> is used, so source code and issue tracker is found there. A list of contributors is also avaible as <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/stargazers/">GitHub profiles list</a>.{% endblocktrans %}</p>
+
+        <p>{% blocktrans %} Project is developed under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. See the <a class="ext-link" href="http://www.gnu.org/licenses/">GNU Affero General Public License</a> for more details.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Unnecessary space at the beginning. Please clean all such things.

File: piplmesh/frontend/templates/about.html [hunk 5]
```diff
@@ -5,6 +5,20 @@
...
+
+        <p>{% blocktrans %}If you have found a bug or have any other technical issue with the site, please use <a href="https://github.com/wlanslovenija/PiplMesh/issues?state=open">GitHub to report it</a>.
+
+All work is done voluntarily, so you are very welcome to help us make it even better. All code is available at <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh">github.com/wlanslovenija/PiplMesh</a>. Read <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/tree/master/docs">docs</a> for instructions.{% endblocktrans %}</p>
+    </div>
+{% endblock %}
+
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Unnecessary line.

File: piplmesh/frontend/templates/about.html (hunk 3 above)
Comment (kubili) [inline piplmesh/frontend/templates/about.html]: This and next line belongs together. I'll fix it.

File: piplmesh/frontend/templates/about.html [hunk 6]
```diff
@@ -5,6 +5,12 @@
 {% block title %}{% trans "About" %}{% endblock %}
 
 {% block content %}
-    <h1>{% trans "About" %}</h1>
-    <p>TODO.</p>
+    <div>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Why this div?

File: piplmesh/frontend/templates/about.html [hunk 7]
```diff
@@ -5,6 +5,12 @@
...
+        <h1>{% trans "About" %}</h1>
+        <p>{% blocktrans %}PiplMesh is the project of students at <a class="ext-link" href="http://www.fri.uni-lj.si/">Faculty of Computer and Information Science</a> of University of Ljubljana and <em>wlan slovenija</em> open wireless network. Its main goal is the development of an open source social networking and info portal for wireless networks used as a welcome page when users connect to the network.{% endblocktrans %}</p>
+        <p>{% blocktrans %}You can read more about the project on  <a class="ext-link" href="http://dev.wlan-si.net/wiki/PiplMesh/">development pages</a> of wlan slovenija.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: Double space.

Comment (mitar) [inline piplmesh/frontend/templates/about.html]: "wlan slovenija" should always be in italics.

File: piplmesh/frontend/templates/about.html [hunk 8]
```diff
@@ -5,6 +5,12 @@
...
+        <p>{% blocktrans %}For development <a class="ext-link" href="https://github.com/">GitHub</a> is used, so source code and issue tracker is found there. A list of contributors is also avaible as <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/stargazers/">GitHub profiles list</a>.{% endblocktrans %}</p>
+        <p>{% blocktrans %}Project is developed under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. See the <a class="ext-link" href="http://www.gnu.org/licenses/">GNU Affero General Public License</a> for more details.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: You can put more links here. Like to the FSF.

File: piplmesh/frontend/templates/about.html [hunk 9]
```diff
@@ -5,6 +5,12 @@
 {% block title %}{% trans "About" %}{% endblock %}
 
 {% block content %}
-    <h1>{% trans "About" %}</h1>
-    <p>TODO.</p>
+    <div>
+        <h1>{% trans "About" %}</h1>
+        <p>{% blocktrans %}PiplMesh is the project of students at <a class="ext-link" href="http://www.fri.uni-lj.si/">Faculty of Computer and Information Science</a> of University of Ljubljana and <em>wlan slovenija</em> open wireless network. Its main goal is the development of an open source social networking and info portal for wireless networks used as a welcome page when users connect to the network.{% endblocktrans %}</p>
+        <p>{% blocktrans %}You can read more about the project on  <a class="ext-link" href="http://dev.wlan-si.net/wiki/PiplMesh/">development pages</a> of wlan slovenija.{% endblocktrans %}</p>
+        <p>{% blocktrans %}For development <a class="ext-link" href="https://github.com/">GitHub</a> is used, so source code and issue tracker is found there. A list of contributors is also avaible as <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/stargazers/">GitHub profiles list</a>.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: This paragraph contains duplicate information as last paragraph. You should somehow resolve this.

File: piplmesh/frontend/templates/about.html [hunk 10]
```diff
@@ -5,6 +5,12 @@
...
+        <p>{% blocktrans %}Project is developed under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. See the <a class="ext-link" href="http://www.gnu.org/licenses/">GNU Affero General Public License</a> for more details.{% endblocktrans %}</p>
+        <p>{% blocktrans %}If you have found a bug or have any other technical issue with the site, please use <a href="https://github.com/wlanslovenija/PiplMesh/issues?state=open">GitHub to report it</a>. All work is done voluntarily, so you are very welcome to help us make it even better. All code is available at <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh">github.com/wlanslovenija/PiplMesh</a>. Read <a class="ext-link" href="https://github.com/wlanslovenija/PiplMesh/tree/master/docs">docs</a> for instructions.{% endblocktrans %}</p>
```
Comment (mitar) [inline piplmesh/frontend/templates/about.html]: No need for precise installation instructions. This they can find when they go to GitHub. No need for two links to GtiHub (here and above).

**Pull request merged**